transports/transport-rest/transport-rest-servlet/src/main/java/io/servicecomb/transport/rest/servlet/RestServletProducerInvocation.java

```diff
@@ -17,20 +17,17 @@
 package io.servicecomb.transport.rest.servlet;
 
 import io.servicecomb.common.rest.RestProducerInvocation;
-import io.servicecomb.common.rest.definition.RestOperationMeta;
 import io.servicecomb.common.rest.filter.HttpServerFilter;
 import io.servicecomb.core.definition.OperationMeta;
 import io.servicecomb.foundation.vertx.http.StandardHttpServletRequestEx;
 
 public class RestServletProducerInvocation extends RestProducerInvocation {
   @Override
-  protected RestOperationMeta findRestOperation() {
```

Comment:
It looks like the new findRestOperation method is just throw the exception if it cannot find the operation.  I'm not sure if this change has some side effect on the user side code.

Reply:
not change, just move from one class to another class.